src/main/modules/PlatformAware.js

```diff
@@ -0,0 +1,47 @@
+const { systemPreferences } = require('electron')
+
+const macConfig = {
```

Comment:
Config objects must be in a function or at least if statements, because otherwise their values are executed, and isDarkMode for instance is macOS specific. It throws on different OSes
```
TypeError: Cannot read property 'isDarkMode' of undefined
```

Assignee: @gtluszcz